tests/metrics/run_docker_metrics

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+
+TIMES=100
+WAIT_TIME=30m
```

Comment:
Not essential, but it would be nice if these variables lived in their own config file (`tests/metrics/run_docker_metrics.dat`?), or could be specified on the command-line.